src/protocols/OutputManagement.cpp

```diff
@@ -306,6 +306,7 @@ COutputConfiguration::COutputConfiguration(SP<CZwlrOutputConfigurationV1> resour
 
         PMONITOR->activeMonitorRule.disabled = true;
         g_pHyprRenderer->applyMonitorRule(PMONITOR, &PMONITOR->activeMonitorRule, false);
+        g_pConfigManager->replaceMonitorRule(PMONITOR->activeMonitorRule);
```

Comment:
you're making it a bool but ignoring the retval. Plus, shouldnt this be first?

Reply:
Yeah about the retval, it looks stupid. The idea was that `replaceMonitorRule` could be used somewhere else in the code, where it might be useful to have the retval. 
I can remove it if you don't like that. Regarding the position, yes probably you're right. In general I would have preferred turning the `activeMonitorRule` to a pointer, but I don't know how much of a good idea that would be (I mean, shouldn't this rule replacing happen also somewhere else?). For now, I'll just swap the position